DesignSafe-CI/portal
# Task/des 1712

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-1712](https://jira.tacc.utexas.edu/browse/DES-1712)

## Summary of Changes: ##
Include project title in breadcrumbs. Expand the PRJ Id's column and wrap the project titles, so that each row is more readable. Make sure that the Workspace data browser matches what is shown in the Data Depot. 

## Testing Steps: ##
1. Check to make sure that Data Depot matches what is show in the Workspace data browser
2. Look to make sure that any long project titles are wrapping and readable in the Workspace data browser
3. Make sure the PRJ Id's are not wrapping in the Workspace data browser
4. Make sure the the Workspace data browser is readable and doesn't require horizontal scrolling
5. Make sure the the Workspace data browser dropdown labels match the side navbar in the Data Depot.

## UI Photos:
Attached to the ticket.

## Notes: ##
